src/views/ConfigEditor.tsx

```diff
@@ -77,6 +77,26 @@ export class ConfigEditor extends PureComponent<ConfigEditorProps> {
             />
           </div>
         </div>
+        <div className="gf-form-group">
+          <h3 className="page-heading">Github Enterprise</h3>
+          <div className="gf-form">
+            <InlineFormLabel className="width-10">Github Enterprise URL</InlineFormLabel>
+            <Input
+              css=""
+              className="width-27"
+              value={jsonData.githubUrl}
+              placeholder="URL of Github Enterprise"
+              summary="URL for Github Enterprise, such as https://github.company.com/api/graphql, leave blank if using github.com"
```

Comment:
Does this have to include `/api/graphql`?

Reply:
It does not have to... updated to include /api/graphql when calling githubv4.NewEnterpriseClient